### DISCUSSION TITLE
Is it possible to search by Nominatim tags in Photon, for example using IATA/ICAO codes for airports?

### DISCUSSION THREAD

**Discussion by dlubom** [komoot/photon#761] [Q&A] [answered]
I have been exploring the Photon geocoder and am interested in its capabilities regarding the usage of specific tags from Nominatim. Specifically, I am looking to search for airports using their IATA or ICAO codes, similar to how it can be done on Nominatim.

For instance, on Nominatim, you can find an airport by searching for its IATA code as shown here:
[https://nominatim.openstreetmap.org/ui/details.html?osmtype=N&osmid=10013516002](https://nominatim.openstreetmap.org/ui/details.html?osmtype=N&osmid=10013516002)
[https://nominatim.openstreetmap.org/ui/search.html?q=WAPF](https://nominatim.openstreetmap.org/ui/search.html?q=WAPF)

Does Photon support this type of tag-based search out of the box? If not, would it be reasonably straightforward to implement this feature? I am considering contributing to the project if this functionality is not currently available but could be added without excessive complexity.

Thank you.

**Comment by lonvia** [comment, marked as answer]
Nominatim considers IATA/ICAO codes a name just like any other. That's why you can find it there. Photon does not copy those names but only standard ones. The list of names to consider is currently hard-coded and changing it not trivial.
